src/sentry/consumers/__init__.py

```diff
@@ -275,6 +275,14 @@ def ingest_events_options() -> list[click.Option]:
             "consumer_type": "events",
         },
     },
+    "ingest-feedbacks": {
+        "topic": settings.KAFKA_INGEST_FEEDBACKS,
+        "strategy_factory": "sentry.ingest.consumer.factory.IngestStrategyFactory",
+        "click_options": ingest_events_options(),
+        "static_args": {
+            "consumer_type": "events",
```

Comment:
I think you might want to make this a new type. We don't really want to run all the `events` logic, and i'm guessing this will diverge in the future?